src/features/profile/components/Header/NonBeneficiaryHeader/NonBeneficiaryHeader.native.test.tsx

```diff
@@ -51,149 +64,168 @@ describe('<NonBeneficiaryHeader/>', () => {
     beforeEach(() => {
       mockNextSubscriptionStep = mockStep
     })
-    it('should render the right banner for 18 years old users, call analytics and navigate to nextBeneficiaryValidationStep', async () => {
-      const setError = jest.fn()
-      const { nextBeneficiaryValidationStepNavConfig } =
-        useBeneficiaryValidationNavigation(setError)
+    it('should render the activation banner for 18 years old users', async () => {
```

Comment:
Le nom du test n'est pas tout à faire vrai, ce serait plutôt "should render the activation banner when user is eligible and api call returns activation banner", un jeune de 15 ans pourra voir cette bannière